src/modules/coreHttpApi/controllers/AttributesController.ts

```diff
@@ -59,6 +61,23 @@ export class AttributesController extends BaseController {
         return this.ok(result);
     }
 
+    @POST
+    @Path("/ExecuteIQLQuery")
+    @Accept("application/json")
+    public async executeIQLQuery(request: any): Promise<Envelope> {
+        return this.ok(await this.consumptionServices.attributes.executeIQLQuery({ query: { queryString: request.query.queryString } }));
```

Comment:
you can pass the request directly to this.consumptionServices.attributes.executeIQLQuery. If you do it like this you can get runtime errors.

Another option would be `this.consumptionServices.attributes.executeIQLQuery({ query: { queryString: request?.query?.queryString }})`